docs/hub/adapters.md

```diff
@@ -0,0 +1,87 @@
+_Adapters_ also provides various methods for composition of adapter modules during training and inference.
+You can learn more about this in the [_Adapters_ paper](https://arxiv.org/abs/2311.11077).
+
+## Exploring _Adapters_ on the Hub
+
+You can find _Adapters_ models by filtering at the left of the [models page](https://huggingface.co/models?library=adapter-transformers&sort=downloads). Some adapter models can be found in the Adapter Hub [repository](https://github.com/adapter-hub/hub). Models from both sources are aggregated on the [AdapterHub website](https://adapterhub.ml/explore/).
+
+## Installation
+
+To get started, you can refer to the [AdapterHub installation guide](https://docs.adapterhub.ml/installation.html). You can also use the following one-line install through pip:
+
+```
+pip install adapters
+```
+
+## Using existing models
+
+For a full guide on loading pre-trained adapters, we recommend checking out the [official guide](https://docs.adapterhub.ml/loading.html). 
+
+As a brief summary, a full setup consists of three steps:
+
+1. Load a base `transformers` model with the `AutoAdapterModel` class provided by _Adapters_.
+2. Use the `load_adapter()` method to load and add an adapter.
+3. Activate the adapter via `active_adapters` (for inference) or activate and set it as trainable via `train_adapter()` (for training). Make sure to also check out [composition of adapters](https://docs.adapterhub.ml/adapter_composition.html).
+
+```py
+from adapters import AutoAdapterModel
+
+# 1.
+model = AutoAdapterModel.from_pretrained("roberta-base")
+# 2.
+adapter_name = model.load_adapter("AdapterHub/roberta-base-pf-imdb")
+# 3.
+model.active_adapters = adapter_name
+# or model.train_adapter(adapter_name)
+```
+
+You can also use `list_adapters` to find all Adapter Models programmatically:
+
+```py
+from adapters import list_adapters
+
+# source can be "ah" (AdapterHub), "hf" (hf.co) or None (for both, default)
+adapter_infos = list_adapters(source="hf", model_name="roberta-base")
+```
+
+If you want to see how to load a specific model, you can click `Use in Adapter Transformers` and you will be given a working snippet that you can load it! 
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-adapter_transformers_snippet1.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-adapter_transformers-snippet1-dark.png"/>
+</div>
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-adapter_transformers_snippet2.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/libraries-adapter_transformers-snippet2-dark.png"/>
+</div>
```

Comment:
The popup window on the hub pages shown in these screenshots ideally would also be updated. How would we do this? Appreciate some help here.

Reply:
You can change the code snippet in https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/library-ui-elements.ts#L38-L43

That said, this change would fail for old `adapter-transformers` models, right?

Reply:
Thanks for the link!

`adapters` is fully backwards compatible, so models created using `adapter-transformers` can be loaded with `adapters` without any changes. We'd like to change the code snippet for all `adapter-transformers` models to promote switching to the new package.

Reply:
That's awesome! If there's backwards compatibility let's do it!